liferay-frontend/liferay-portal
# LPS-155249 [Content Dashboard] Hover actions need tooltips

[Jira ticket](https://issues.liferay.com/browse/LPS-155249)

## Motivation
The quick actions offered by the Dropdown component, used in the Content Dashboard module, offer no tooltip on hover

## Solution proposed
Add title prop, using the existent label value

## Steps to validate
- Create a new site, ie. MasterClass, just to populate the CD with some content
- Navigate to Control Menu > Content > Content Dashboard
- Hover a table row, quick actions should appear at the right of the row
- Hover a quick action, ie. the pencil icon

### Expected:
- Each quick action has a tooltip
- The pencil icon should display an tooltip with "Edit" content text

### Actual result: No tooltip

This PR tries to address this problem.

## Advice

As far as I know, the quick actions are only used in Content Dashboard, so this change should have no impact in other usages of the taglib 🤔 

## Images

![image](https://user-images.githubusercontent.com/19485114/171847994-298d79af-627f-41c7-8004-a5cc528d3398.png)
